java/core/src/main/java/com/whylogs/core/metrics/IntegralMetric.java

```diff
@@ -0,0 +1,99 @@
+package com.whylogs.core.metrics;
+
+import com.whylogs.core.PreProcessedColumn;
+import com.whylogs.core.SummaryConfig;
+import com.whylogs.core.metrics.components.MaxIntegralComponent;
+import com.whylogs.core.metrics.components.MinIntegralComponent;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+
+@Getter
+@EqualsAndHashCode(callSuper=false)
+public class IntegralMetric extends BaseMetric<IntegralMetric> {
```

Comment:
This should be `IntsMetric` to match Python

Reply:
Got it :)

Reply:
I'm going to make the formal ask that the python moves to IntegralMetrics as 'IntsMetric' feels very unclear to me. Is it a list of ints, is it just for the int type, and what on earth does it do? Integral isn't a big change but it stays in line with our IntegralMetricComponent, doesn't sound like a list, and (at least coming from Tableau) makes sense in industry why there would be a min/max to it. :) @richard-rogers @jamie256

Reply:
I would have preferred "Integer" and "Rational" over "Integral" (we're not doing calculus ;) and "Fractional". But alas, we've shipped "IntsMetric" so I think the best we could do is deprecate it and make `IntegralMetric` an alias for it. `MinMaxMetric` might be better?

Reply:
That would be better. :) that would be lovely down the road. :)